### ISSUE TITLE
available_payment_methods empty in GraphQL with version 1.18.0

### ISSUE THREAD

**Issue by bramstroker** [mollie/magento2#316]
**Describe the bug**
When using the new version of the module no available payment methods are returned when using GraphQL

**Used versions**
- Magento 2.4.1 Community edition
- Mollie 1.18.0

**To Reproduce**
1. Fill in a correct Mollie API key and enable the Ideal payment method for the given store.
2. Create an empty cart using GraphQL:
```
mutation {
  createEmptyCart
}
```

3. Query for available payment methods using GraphQL:
```
query ($cart_id: String!) {
  cart(cart_id: $cart_id) {
    available_payment_methods {
      code
    }
  }
}
```

4. No payment methods are returned. When I put a product in the cart I have the same issue.

**Expected behavior**
Possible Mollie payment methods are returned correctly

**Actual behavior**
No mollie payment methods are returned. 
An error is logged `[2020-11-10T10:10:00+0000] Error executing API call (400: Bad Request): The currency is missing from the amount. Field: amount.currency. Documentation: https://docs.mollie.com/guides/handling-errors`.
The root cause is in https://github.com/mollie/magento2/blob/7ba6aeee18ee231c1f7a75c643aa77e880f548b1/Model/MollieConfigProvider.php#L228. The quote model which is returned by the checkout session is a empty quote, this is caused by the checkout session having no quoteId set. Because the quote model returned by the checkout session has no data the currency could not be retrieved from the quote resulting in this exception.
I have the same issue on different environments. The GraphQL checkout flow is working without any problems with version 1.17.1 of the Mollie module

**Comment by royduin** [comment]
Same problem with the REST API.

**Comment by Frank-Magmodules** [comment]
Hi @bramstroker & @royduin , 
Thank you for opening the issue. We've been able to reproduce this and have placed this on our backlog. We will work on a solution and release it in an upcoming release. Thank you for your contribution to this.

**Comment by Frank-Magmodules** [comment]
Hi @bramstroker & @royduin ,
We are happy to share that we've just released the new [1.19.0](https://github.com/mollie/magento2/releases/tag/v1.19.0) version with a fix for this issue. Thank you again for the detailed reports and patience. We are closing this issue now but please feel free to reopen the issue if this still occurs.
